### PR TITLE
Dlg factor data frame

### PR DESCRIPTION
**Pull request by stevenndungu**

### CODE REVIEW

File: dlgFactorDataFrame.vb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿Imports Microsoft.VisualBasic
+
+Public Class Class1
+
+End Class
```
Comment (dannyparsons) [inline dlgFactorDataFrame.vb]: Can this be deleted?

File: instat/dlgFactorDataFrame.vb [hunk 2]
```diff
@@ -1,42 +1,3 @@
...
-        ucrBase.iHelpTopicID = 162
-
-    End Sub
-    Private Sub SetDefaults()
-
-    End Sub
-
-    Private Sub ucrBase_ClickReset(sender As Object, e As EventArgs) Handles ucrBase.ClickReset
-        SetDefaults()
-    End Sub
```
Comment (dannyparsons) [inline instat/dlgFactorDataFrame.vb]: Was there a reason to get rid of this code?

**Comment by stevenndungu** [comment]
Ooh sorry! It was as a result of confusion I didn't actually notice. while
with the intention of creating a form with name dlgFactorDataFrame,
something different come up and so I had to redo the process of creating
it. I opted to replace it - and that's where the whole problem resulted.

Fixing it.

**Comment by stevenndungu** [comment]
*fixed it

**Pull request closed without merging**